cmd/nakedret/main.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"golang.org/x/tools/go/analysis/singlechecker"
+
+	"github.com/alexkohler/nakedret"
+)
+
+const (
+	DefaultLines = 5
+)
+
+func main() {
+	analyzer := nakedret.NakedReturnAnalyzer(DefaultLines)
+	singlechecker.Main(analyzer)
+}
```

Comment:
moved `main()` to `cmd/nakedret/main.go` so `nakedret` code can use `package nakedret` and be imported